<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetWeb</class>
 <widget class="QWidget" name="widgetWeb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Web Services</string>
  </property>
  <property name="windowIcon">
   <iconset resource="web.qrc">
    <normaloff>:/icons/web.png</normaloff>:/icons/web.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="pushRefresh">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboServices">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Select...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushBack">
        <property name="text">
         <string>Back</string>
        </property>
        <property name="icon">
         <iconset resource="web.qrc">
          <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>136</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushForward">
        <property name="text">
         <string>Forward</string>
        </property>
        <property name="icon">
         <iconset resource="web.qrc">
          <normaloff>:/icons/arrow-right.png</normaloff>:/icons/arrow-right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushReload">
        <property name="text">
         <string>Reload</string>
        </property>
        <property name="icon">
         <iconset resource="web.qrc">
          <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushPrint">
        <property name="text">
         <string>Print</string>
        </property>
        <property name="icon">
         <iconset resource="web.qrc">
          <normaloff>:/icons/document-print.png</normaloff>:/icons/document-print.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushBrowser">
        <property name="text">
         <string>View In Browser</string>
        </property>
        <property name="icon">
         <iconset resource="web.qrc">
          <normaloff>:/icons/web.png</normaloff>:/icons/web.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Service:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="web.qrc"/>
 </resources>
 <connections/>
</ui>
